paddle/phi/api/lib/CMakeLists.txt

```diff
@@ -97,6 +97,26 @@ set(wrapped_infermeta_header_file
 set(wrapped_infermeta_source_file
     ${CMAKE_SOURCE_DIR}/paddle/phi/infermeta/generated.cc)
 
+# tensor and tensor operants file
+set(tensor_gen_file
+    ${CMAKE_SOURCE_DIR}/paddle/phi/api/yaml/generator/tensor_gen.py)
+set(api_prim_yaml_file ${CMAKE_SOURCE_DIR}/paddle/fluid/prim/api/api.yaml)
```

Comment:
This file will be deleted in [[Decouple] Decouple prim/api/api.yaml in phi CMakeLists #50595](https://github.com/PaddlePaddle/Paddle/pull/50595)